tests/attestation_plugin/enc/enc.c

```diff
@@ -271,66 +243,24 @@ static void _test_sgx_remote()
             &evidence_size),
         OE_OK);
 
-    verify_sgx_evidence(
-        &_ecdsa_report_uuid,
-        false,
-        evidence,
-        evidence_size,
-        NULL,
-        0,
-        endorsements, // reuse the endorsement generated above
-        endorsements_size,
-        test_claims,
-        TEST_CLAIMS_SIZE);
-
-    OE_TEST(oe_free_evidence(evidence) == OE_OK);
-    OE_TEST(oe_free_endorsements(endorsements) == OE_OK);
-
-    printf("testing a 65-byte custom claims that should failed\n");
-    OE_TEST_CODE(
-        oe_get_evidence(
+    {
+        static const oe_uuid_t _ecdsa_report_uuid = {
+            OE_FORMAT_UUID_LEGACY_REPORT_REMOTE};
+        verify_sgx_evidence(
```

Comment:
I think we still need a test to verify those endorsement-related claims can be correctly extracted from the `oe_report` and `sgx_quote`. That's a typical MAA user case. Is there any way to reuse the endorsement generated above?

Reply:
Agreed. Will find a way to add endorsements check.